### DISCUSSION TITLE
How to modify the height of window title bar?

### DISCUSSION THREAD

**Discussion by hungtcs** [tauri-apps/tauri#7978] [Q&A]
Macos has several different window types, their title bars have different sizes, for UI design purposes I have to use a loose layout, 
 but I didn't find the relevant configuration that can be used to modify it, can anyone help me?

1. This is the style I want, the buttons at the back can be aligned with the traffic lights

    <img width="198" alt="image" src="https://github.com/tauri-apps/tauri/assets/11692935/f87ab73d-2eea-4d8b-9b17-ab3bdffcd717">

2. This is my actual style, I can't adjust the position of the traffic light, its location is too close to the top

    <img width="249" alt="image" src="https://github.com/tauri-apps/tauri/assets/11692935/55f1cf8b-86a2-485e-be93-17f9c7d0c447">

Thanks.

**Comment by liyasthomas** [comment]
Has anyone found any solutions to align traffic lights with custom window title?

**Reply by FabianLars** [reply]
https://github.com/tauri-apps/tauri/pull/12366 ?

**Reply by TeemuKoivisto** [reply]
@FabianLars amazing, it works like a charm! Here's an example config for others wandering here:

```
      {
        "width": 800,
        "height": 600,
        "hiddenTitle": true,
        "titleBarStyle": "Overlay",
        "trafficLightPosition": { "x": 15, "y": 20 }
      }
```
